### DISCUSSION TITLE
Who's hiring? (Apr 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#35840] [Ideas]
# What is this?
New month, new post! This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

*If you're looking for last month's discussion, see #34949. At least one company has [hired someone](https://twitter.com/Nutlope/status/1475929984174895105) as a result of it!* 

If you hire someone through this job board, please reach out to me (hassan@vercel.com) so I can send you some swag :)

If you're a developer looking to post about themselves, use #35839.

**Comment by btslater** [comment]
**ResearchRabbit | Founding Front-End Engineer (React+Redux) | Remote (US-based) | Full-Time | $125,000—$175,000 salary + 0.5—1.5% equity | [LINK](https://www.notion.so/researchrabbit/Founding-Front-End-Engineer-React-Redux-3f357f5fefae485a84d403acd648f734)**

_ResearchRabbit is building the "OS" for the knowledge economy - on a mission to empower knowledge workers around the world!_

We are looking to hire a **Founding Front-End Engineer** with excellent skills in React and Redux. If you are looking for fun and engaging challenges on a platform that involves novel approaches to working with and visualizing complex data, please check out our full job description and apply!

**Location**: Remote (US timezones-ish)

**Salary**: $125,000 - $175,000

**Equity**: 0.5-1.5%

_(For particularly exceptional candidates, we are willing to negotiate beyond these ranges)_

[Apply Here!](https://www.notion.so/researchrabbit/Founding-Front-End-Engineer-React-Redux-3f357f5fefae485a84d403acd648f734)

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for May is up (https://github.com/vercel/next.js/discussions/36738) and you're more than welcome to repost this message there :)

**Comment by sullivan-sean** [comment]
Fig | Frontend Engineer | US (Ideally Bay Area) | Full-time | Competitive Salary + Equity Comp | [Link](https://fig.io/jobs/frontend)

Hey everyone! 

I'm Sean, founding engineer @ Fig ([fig.io](http://fig.io)). Fig makes developers more productive in the terminal. 

- **Mission**: re-imagine the terminal. First, autocomplete, then a fully-fledged App Ecosystem 
- **Why**: 50M+ engineers use the terminal. It has barely changed since the 70s.
- **Investors**: Y Combinator, General Catalyst, Kleiner Perkins, founders/execs at Stripe, GitHub, Heroku, Segment
- **Traction**: One of the fastest growing OSS startups in 2021; our users love us: see twitter.com/fig

We're looking for frontend engineers experienced with Next & React to come join us and help lead our new product 🤫

If you're interested, please reach out to me (sean@fig.io) or apply at https://fig.io/jobs/frontend

**Comment by jordie23** [comment]
CarExpert.com.au | Full-stack Developer | Remote (Australia only), or Sydney, Melbourne or Brisbane | Full-time | AUD$110-140k base | [Link](https://www.seek.com.au/job/56488454)

[CarExpert](https://www.carexpert.com.au/) is a fast growing startup and we're looking for more developers to help us keep development up with our ideas. Our mission is to be the most trusted experts in car ownership, enabling confident decisions, so we are working on innovative solutions that will make a new car buyer's journey a fun and stress-free experience.

I'm the CTO. We have a team consisting of myself and other 4 full-time developers spread across Australia. We're looking for another full-time full-stack developers. Daily work will vary between NextJS and our AWS backend. We're currently implementing new parts of our site using KeystoneJS CMS.

At the moment all our developers are working remotely in various locations in Australia. You're welcome to WFH full time, or come into one of our offices if you live in Syd, Melb or Bris. You must have the right to work in Australia. Please use the [seek link](https://www.seek.com.au/job/56488454) for more info and to apply.

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for May is up (https://github.com/vercel/next.js/discussions/36738) and you're more than welcome to repost this message there :)

**Comment by amandahillPDI** [comment]


**Reply by abe312** [reply]
How is this relevant to next.js? It's not even a React position! 🤦🏻‍♂️

**Reply by Nutlope** [reply]
Thanks for letting us know @abe312, I'm removing it

**Comment by spencermarx** [comment]
### [LaFabrica](https://www.lafabrica.work/) | Full-Stack Engineers | Remote (Worldwide) | Full-time & Part-Time | [Link](https://www.lafabrica.work/developers/register)

We are looking for Fullstack Engineers to join our platform, [LaFabrica](https://www.lafabrica.work/), a crypto-driven cooperative for Software Engineers.

**The platform...**
1. Connects developers across the world directly to remote software projects paid in US Dollars 💵
2. Offers members achievement-based dividends, distributed using our very own cryptocurrency 🏦
3. Provides startups and dev agencies dedicated, thoroughly vetted, ready-to-work remote engineering teams and individual developer resources within a range of budgets 👩‍💻👨‍💻

Our Beta pool is filling up quickly, so if you’re interested, sign up soon!

Registration: ➡️ [here](https://www.lafabrica.work/developers/register) ⬅️

**Our Stack:**
Completely flexible -- from full stack Javascript to GoLang, we encourage devs from any stack to apply.

The current platform is built out using:
* Typescript
* NextJS | React | Material UI
* NestJS | Firebase
* AWS Lambda | Google Cloud Functions

Feel free to apply using the link above!

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for May is up (https://github.com/vercel/next.js/discussions/36738) and you're more than welcome to repost this message there :)

**Comment by cdaringe** [comment]
**walmart | full-stack web (various roles) | remote | Full-time | $100-200k base (+ awesome benefits/RSUs) | [careers.walmart.com](https://careers.walmart.com/)**

Greetings all. We have a variety of positions open (engineers, manager, and TPM) on the walmart web platform team. Your role may include any or all of the following:

- Build devtools. Help our community of teams get their work done, while keeping our solutions lightweight and effective
  - Completed projects in the last week were integrating with our translation management service, GraphQL query optimizations/compilations, adding VSCode tasks to automate common workflows.
- Develop the core of 5+ next.js applications, support 500+ typescript libraries. These numbers will grow n' grow
- Help manage a community.
  - We have 400+ engineers in a large typescript + next + nx monorepo
  - Design codified & documented processes and/or constraints that balance community performance with team freedoms/flexibility
- Think hard, execute thoughtfully. Quality and project scalability are challenges in any project, but exacerbated when you have a large diverse community of talent and backgrounds, all racing ahead on feature work
- Optimize processes. CI, scripts, compilers--whatever. Find the slow stuff, make it faster where you can.
- Test everything. Unit, integration, and functional tests. We're aiming for Continuous Delivery, and have the mind share and toolkit to get there. Help us get over the line!
- ...more. Being on the platform team, you also have to partner w/ our critical operational partners often. But that's a bit meta for this post 😄 

It's a very unique position. High coding, high collaboration. Beyond normal agile-ish rhythms, we meet 3x a week to structurally break down current or existential challenges. You'll enjoy it.

What we're looking for.

- required: node.js & browser excellence. if you're strong in just one, still talk to us
- required: patience. thoughtfulness. slow speaking, friendly, clear communicators
- strongly desired: typescript-fu
- desired: experience with steering a community or group

Send me an email @ christopher.dieringer@walmart.com with `[HIRING, NEXT]: Web Platform Team - <Your Name>` as the email subject. Give me your self-evaluated skill level (junior engineer, mid-level, senior, some-thing-above :)) and I'll reply back with an application to fill out. I'm still collecting our application links :). I'll update this post when I get them, so you may be able to dodge the email outright shortly!

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for May is up (https://github.com/vercel/next.js/discussions/36738) and you're more than welcome to repost this message there :)

**Comment by Silvia2801** [comment]
HUAWEI TECHNOLOGY AG | JUNIOR NETWORK ENGINEERS | ZURICH | FULL TIME | PAID INTERNSHIP + GRADUATE PROGRAM (perm. contract)
We are looking for talented graduates who are eager to develop their careers at a global leading Telecommunication Company. Currently, we have 5 vacancies:

**Junior IP design & integration engineer
Junior Core Network Engineer Roaming
Junior Core Network Engineer DB Mobile
Cyber Security Specialist
Junior RF Engineer**

Your Profile
Recently graduated (2021-2022) with a bachelor’s or higher degree in telecommunications, technology, computer science, or another related major. High IT affinity: coding and scripting is your passion. Understanding 2G/3G/LTE basic Technology. Understanding of TCP/IP Protocol. Hands-on mentality, eager to learn and develop yourself continuously. Love to work in a diversified and cross-culture organization
**Candidates must be eligible to work in Switzerland**
We Offer
3 months of training through our Internship program, afterward we offer a permanent contract within the Career Graduate Program.
If you are interested, send me an email @ silvia.dangelo@huawei.com

**Reply by Nutlope** [reply]
Hey! Just want to let you know that the new job board for May is up (https://github.com/vercel/next.js/discussions/36738) and you're more than welcome to repost this message there :)
